core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -596,6 +596,11 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
    */
   final def interruptStatus(flag: InterruptStatus): ZIO[R, E, A] = new ZIO.InterruptStatus(self, flag)
 
+  /**
+    * Ignores any failure and return `Unit`
+    */
+  final def ignore: ZIO[R, Nothing, Unit] = self.either.unit
```

Comment:
Looks great! Be sure to place it alphabetically in the list of methods. 👍

Reply:
The ZIO functions are not ordered, I changed the `ignore` place but I'm not sure.